dockerfiles/debian-bullseye

```diff
@@ -26,6 +26,11 @@ ENV JAVA_HOME=/opt/java/openjdk
 COPY --from=eclipse-temurin:11 $JAVA_HOME $JAVA_HOME
 ENV PATH="${JAVA_HOME}/bin:${PATH}"
 
+# These are needed for the Clouseau integration
```

Comment:
We also recently added bookworm, let's add it to that as well so all the versions have it